README/ReleaseNotes/v614/index.md

```diff
@@ -47,6 +48,7 @@ The following people have contributed to this new version:
    - Implement reading of objects data from JSON
    - Provide TBufferJSON::ToJSON() and TBufferJSON::FromJSON() methods
    - Provide TBufferXML::ToXML() and TBufferXML::FromXML() methods
+   - LZ4 is enabled as a default ROOT compression algorithm (with compression level 4)
```

Comment:
I would move this as the first bullet.  Consider the phrasing "LZ4 (with compression level 4) is now the default compression algorithm for new ROOT files."  and Consider adding a short explanation of the benefit of LZ4.  thanks.